README.md

```diff
@@ -204,6 +204,7 @@ provide the path to a JAR file including an SPI implementation using the system
 | [khttp](https://khttp.readthedocs.io)                                                                                                 | 0.1+                           |
 | [Kubernetes Client](https://github.com/kubernetes-client/java)                                                                        | 7.0+                           |
 | [Lettuce](https://github.com/lettuce-io/lettuce-core)                                                                                 | 4.0+                           |
+| [Log4j 2](https://logging.apache.org/log4j/2.x/)                                                                                      | 2.13.2+                        |
```

Comment:
We generally try to support as old version as possible. Are there some hard reasons why we cannot decrease this version?

Reply:
A, I see. Our current manual instrumentation uses a recent SPI. But if we go to bytecode level, can we actually provide support for older versions?

Reply:
> But if we go to bytecode level, can we actually provide support for older versions?

Yes, we can. I can add a separate instrumentation and instrument all `ContecxtDataInjector` implementations and support as low as 2.7 (which is 4 years old). If we want to go lower it'd probably turn out to be more complicated.

Reply:
Done.